examples/terraform/azure/main.tf

```diff
@@ -127,7 +130,6 @@ resource "azurerm_lb" "lb" {
 }
 
 resource "azurerm_lb_backend_address_pool" "backend_pool" {
-  resource_group_name = azurerm_resource_group.rg.name
```

Comment:
Why is it needed to remove this?

Reply:
If we keep this, we get a deprecation notice for this attribute and that it will be removed in future versions of terraform.